apps/chat/src/components/Marketplace/TabRenderer.tsx

```diff
@@ -194,6 +196,15 @@ export const TabRenderer = ({ isMobile }: TabRendererProps) => {
     [setDetailsModel],
   );
 
+  useEffect(() => {
+    if (!detailsModel) return;
+    setDetailsModel(
```

Comment:
Lets update `detailsModel` inside `handleEdit` method

Reply:
here is the entity before edit